Microsoft.CAP.Portal/Microsoft.CAP.Portal/Views/Home/Index.cshtml

```diff
@@ -1,31 +1,118 @@
+                    <div class="col-xs-8">
+                        <input class="form-control" id="fromDatePicker" type="text" />
+                    </div>
+                </div>
+                <div class="form-group" style="padding-bottom:2em">
+                    <label class="col-xs-4" for="toDatePicker">To:</label>
+                    <div class="col-xs-8">
+                        <input class="form-control" id="toDatePicker" type="text" />
+                    </div>
+                </div>
+                <button class="btn btn-primary col-xs-offset-2 col-xs-8" id="btn_getHistory">Get History Result</button>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<!--tuning-->
+<div class="row vertical-align" style="margin-top:2em">
+    <div class="col-md-8" id="chart2">
+        <img src="~/Images/loading2.png" alt="loading" style="width:100%;position:relative" />
+        <div class="hidden-xs" style="position:absolute;top:50%;left:50%;transform:translate(-50%,-50%)">
+            <h3 style="font-family:Consolas;color:ghostwhite">
+                1. After getting History Result, modify the Parameters on the right;<br />
+                2. Retrain and you will get a Comparison Chart;<br />
+                3. If you are satisfied with the Tuning Result, please click on the Save Button
+            </h3>
+        </div>
     </div>
     <div class="col-md-4">
-        <h2>Web Hosting</h2>
-        <p>You can easily find a web hosting company that offers the right mix of features and price for your applications.</p>
-        <p><a class="btn btn-default" href="http://go.microsoft.com/fwlink/?LinkId=301867">Learn more &raquo;</a></p>
+        <div class="panel panel-success">
+            <div class="panel-body">
+                <!--parameters example-->
+                @foreach (var item in Model.Parameters)
+                {
+                    <div class="form-group" style="padding-bottom:2em">
+                        <label class="col-xs-4" for="@item.Key">@item.Key:</label>
+                        <div class="col-xs-8">
+                            <input class="form-control" id="@item.Key" type="text" value="@item.Value" />
+                        </div>
+                    </div>
+                }
+                <button class="btn btn-success col-xs-offset-2 col-xs-8" id="btn_tuning">Retrain</button>
+                <button class="btn btn-warning col-xs-offset-2 col-xs-8" style="margin-top:2.5em" id="btn_save">Save</button>
+            </div>
+        </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+<script>
+    $(document).ready(function () {
+        //default:yesterday
+        var yesterday = new Date(new Date() - 24 * 60 * 60 * 1000);
+        $("#fromDatePicker").val((yesterday.getMonth()+1) + "/" + yesterday.getDate() + "/" + yesterday.getFullYear());
+        $("#toDatePicker").val((yesterday.getMonth()+1) + "/" + yesterday.getDate()+"/"+yesterday.getFullYear());
+
```

Comment:
How about first define a var dateString = (yesterday.getMonth()+1) + "/" + yesterday.getDate() + "/" + yesterday.getFullYear(), instead of wriote it twice